midealocal/devices/db/__init__.py

```diff
@@ -82,10 +234,52 @@ def process_message(self, msg: bytes) -> dict[str, Any]:
         ]
         for status in self._attributes:
             if hasattr(message, str(status)):
+                value = getattr(message, str(status))
+                # parse progress
                 if status == DeviceAttributes.progress:
-                    self._attributes[status] = progress[getattr(message, str(status))]
+                    self._attributes[status] = progress[value]
+                # parse status
+                elif status == DeviceAttributes.status:
+                    if value in MideaDBDevice._status:
+                        self._attributes[DeviceAttributes.status] = (
+                            MideaDBDevice._status.get(value)
+                        )
+                    else:
+                        self._attributes[DeviceAttributes.status] = None
```

Comment:
```suggestion
                    self._attributes[DeviceAttributes.status] = (
                         MideaDBDevice._status.get(value)
                        )
```

The purpose of `get()` is exactly to avoid the if

Reply:
value may not list in _status, just return None.

Reply:
?

Reply:
it's a very simple logic, if value not in list, we can't return error, just set the status to None.
so why you thinks the value always in _status ? we just list what we want to display, and it's not a full list with different sub modle for this device.
if there is any issue ,please add your comments, thanks

Reply:
fixed with comment comments